docs/content/en/index.md

```diff
@@ -8,7 +8,16 @@ category: ''
 <img src="/preview.png" class="light-img" width="1280" height="640" alt=""/>
 <img src="/preview-dark.png" class="dark-img" width="1280" height="640" alt=""/>
 
-[Sentry Module](https://github.com/nuxt-community/sentry-module) for [NuxtJS](https://nuxtjs.org).
+[Sentry Module](https://github.com/nuxt-community/sentry-module) for [NuxtJS 2](https://v2.nuxt.com/).
+
+
+<alert type="warning">
+
+  If you want support for Nuxt 3, please, raise your opinion [in this discussion](https://github.com/getsentry/sentry-javascript/discussions/6929) and ensure to make clear that the Nuxt community would love to see a first-class support from Sentry 🔥.
```

Comment:
I don't think they will appreciate people who are adding "me too" comments. It should be enough to show support by adding reaction on the first post there.